src/modules/auth/actions/login-with-uport.js

```diff
@@ -1,18 +1,16 @@
+import { augur } from 'services/augurjs'
 import { updateIsLogged } from 'modules/auth/actions/update-is-logged'
 import { loadAccountData } from 'modules/auth/actions/load-account-data'
-import { augur } from 'services/augurjs'
-import uPortSigner from 'modules/auth/helpers/uport-signer'
+import { uPortSigner } from 'modules/auth/helpers/uport-signer'
 
-export default function loginWithUport(account, signingFunction) {
-  return (dispatch) => {
-    dispatch(updateIsLogged(true))
-    dispatch(loadAccountData({
-      ...account,
-      meta: {
-        address: account.address,
-        signer: txObj => uPortSigner(txObj, dispatch),
-        accountType: augur.rpc.constants.ACCOUNT_TYPES.U_PORT,
-      },
-    }))
-  }
+export const loginWithUport = account => (dispatch) => {
```

Comment:
Why was this changed from being an `export default`? I thought we were trying to use `export default` whenever possible.

Reply:
Just for consistency, generally export const is used else where.